lib/modules/saveComments.js

```diff
@@ -39,10 +54,17 @@ const savedCommentStorage = Storage.wrapBlob('RESmodules.saveComments.savedComme
 |} => { throw new Error('Saved comment not found'); });
 let savedCommentIDs;
 
+
 module.beforeLoad = async () => {
 	savedCommentIDs = new Set(Object.keys(await savedCommentStorage.getAll()));
 
 	watchForThings(['comment'], addSaveLinkToComment);
+	if (module.options.enableCommentSaveOnReply.value) {
+		watchForThings(['comment'], addReplyListener);
+	}
+	if (module.options.enableInlinePreviousComment.value) {
+		watchForThings(['comment'], addPreviousCommentButton);
```

Comment:
I think it would be noisy to have this button added to all comments regardless of whether they have been saved. What do you of extending the `saveElement` button to show the previous comment?